Various/ausbaxter_Render item columns.lua

```diff
@@ -0,0 +1,3878 @@
+--@description ausbaxter_Render item columns
```

Comment:
Please don't repeat the author name in the package name. This clutters the list and breaks alphabetical sorting.